src/Games/NexusMods.Games.StardewValley/Diagnostics.cs

```diff
@@ -34,10 +49,22 @@ internal static partial class Diagnostics
     internal static IDiagnosticTemplate RequiredDependencyIsOutdatedTemplate = DiagnosticTemplateBuilder
         .Start()
         .WithId(new DiagnosticId(Source, number: 2))
-        .WithTitle("Required dependency is outdated")
+        .WithTitle("Outdated Dependency")
         .WithSeverity(DiagnosticSeverity.Warning)
-        .WithSummary("Mod {Dependent} requires at least version {MinimumVersion} of {Dependency} but installed is {CurrentVersion}")
-        .WithDetails("You can download the latest version at {NexusModsDependencyUri}")
+        .WithSummary("'{Dependent}' requires an updated version of '{Dependency}'")
+        .WithDetails("""
+The mod **{Dependent}** requires **{Dependency}** version {MinimumVersion} or higher to function, but an older version of **{Dependency}** ({CurrentVersion}) is installed.
+
+### How to Resolve
+1. Download the latest version of **{Dependency}** from {NexusModsDependencyUri}
+2. Add the latest version of **{Dependency}** to the loadout
```

Comment:
Should say to download version {MinimumVersion} or greater instead perhaps?

Reply:
MinimumVersion is not exposed as part of this message.

Reply:
https://github.com/Nexus-Mods/NexusMods.App/pull/2690/commits/fc642e0c1d5363cc8a041faa30744dcfb354b79c